setup/providers/openstack.md

```diff
@@ -9,6 +9,95 @@ sidebar:
 
 ## Prerequisites
 
+The Spinnaker Openstack driver is developed and tested against the Openstack Mitaka release.
+Due to the limitless ways to configure Openstack and its services,
+here is a list of API versions that are required to be enabled:
+
+* Keystone (Identity) v3
+* Compute v2
+* LBaaS v2
+* Networking v2
+* Orchestration (Heat)
+* Ceilometer
+* Glance v2
+
+You will need an account admin permissions for Spinnaker to use. You can download the [openrc](https://docs.openstack.org/user-guide/common/cli-set-environment-variables-using-openstack-rc.html) from your Horizon UI. To test your setup, use the the [OpenStack command line client](https://docs.openstack.org/developer/python-openstackclient/).
+
 ## Adding an Account
 
+First, make sure that the provider is enabled:
+
+```bash
+hal config provider openstack enable
+```
+
+The OpenStack account information is in your openrc.sh. Run the following `hal` command to add an account named `my-openstack-account` to your list of OpenStack accounts, filling your specific information as needed:
+
+
+```bash
+hal config provider openstack account add my-openstack-account \
+    --auth-url http://authurl:5000/v3  --username service-account --domain-name default --regions RegionOne --project-name the-project --password service-password
```

Comment:
Could you split these parameters into multiple lines using `\`? It'll make it a little easier to copy/paste.